test/test_ask.py

```diff
@@ -74,3 +74,24 @@ def test_parse_video_response():
         }
     ]
     assert resp == expected_resp
+
+
+def test_search_ask_without_count():
+    query = 'fossasia'
+    expected_max_resp_count = 10
+    resp_count = len(Ask().search(query))
+    assert resp_count <= expected_max_resp_count
```

Comment:
With a popular search term like _fossasia_ we should get all of the search results that we ask for even if we ask for 27.  My worry with the current test is that the scraper could return zero hits and still pass the test.

Reply:
Quora returns only 9 results for "fossasia" and in general, the maximum response count for quora is 10. I don't know about the results of other search engines. So, should I reduce the max_count(27) and check for "==" and have different tests for quora?

Reply:
Keep the high maxcount but keep the <= where it _needs_ to be but explain that choice in a comment.

Reply:
Sorry for the late reply. I will make the changes by tomorrow.

Reply:
@cclauss I have added the necessary changes, but can you help me understand why the travis ci build is failing for python2?